### PR TITLE
Set "errexit" and explicitly check for root privileges

### PR DESCRIPTION
**Pull request by mojotx**
This change will set the "errexit" option so that script execution fails if there is an error. 

It will also check to see if the user has root privileges, and if not, will generate an error message an exit.

### CODE REVIEW

File: enable-ius.sh [hunk 1]
```diff
@@ -3,6 +3,12 @@
 # Script to setup the IUS public repository on your EL server.
 # Tested on CentOS/RHEL 6/7.
 
+set -o errexit
```
Comment (carlwgeorge) [inline enable-ius.sh]: As this is an alias for `set -e`, let's just use `set -e` directly instead.

File: enable-ius.sh [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+if [[ ${UID} -ne 0 && ${EUID} -ne 0 ]]; then
```
Comment (carlwgeorge) [inline enable-ius.sh]: `EUID` is only different from `UID` when a non-root user executes a file owned by root with the setuid bit.  I don't see that as a realistic scenario for this script, so let's check for `UID` only.

**Review by carlwgeorge — CHANGES_REQUESTED** [review]
Thanks @mojoTX for the suggested improvement.

File: enable-ius.sh (hunk 2 above)
Comment (mojotx) [reply enable-ius.sh]: Done.

File: enable-ius.sh (hunk 1 above)
Comment (mojotx) [reply enable-ius.sh]: Done. I normally prefer `set -e` myself, but I thought being more verbose might be preferred for sample code.

**Review by mojotx — COMMENTED** [review]
I've made the suggested changes. I'm not sure if I need to close this pull-request, and do a new one, or if this one will still work since I've pushed another commit.

**Review by carlwgeorge — APPROVED** [review]

**Comment by carlwgeorge** [comment]
No need for a new pull request, when you push additional commits to your compare branch it automatically updates the pull request.  GitHub even sends notifications for it to everyone subscribed to the pull request.

The changes are live, thanks for contributing to IUS!

**Pull request merged**